core/tests/test_utils.py

```diff
@@ -1549,6 +1549,7 @@ def save_new_skill(
         skill = skill_domain.Skill.create_default_skill(skill_id, description)
         if misconceptions:
             skill.misconceptions = misconceptions
+            skill.next_misconception_id = len(misconceptions) + 1
```

Comment:
Somewhere we check that the misconception IDs in the incoming object are unique, right?

Reply:
We don't explicitly check that, but the misconception ID is automatically created each time a new misconception is created based on `next_misconception_id`.
Then, there is a check that checks whether any misconception id is greater than `next_misconception_id`, and raises an error, if so.

Reply:
But are you relying on behaviour in the frontend here, or is the behaviour you describe occurring in the backend? If the former we still need a validation check because frontend data is untrusted.

Reply:
Done, added a check.